pages/qci_ui_tools.py

```diff
@@ -0,0 +1,44 @@
+from selenium.webdriver.common.by import By
+
+
+class QciUiTools():
```

Comment:
Yes.

I discussed it in the commit message:  https://github.com/RedHatQE/fusor-qe/pull/58/commits/484f8e6663353868d125e5c9500d2336db4c4e0f

However to restate, up to this point there hasn't been any QCI specific stuff in Base or Page and I was trying to put shared QCI specific stuff in it's own library.   I think over time having that sort of separation will make it easier to re-use some of the code that is not QCI specific.

Another thing to be careful of too, is having classes that one inherits from which are essentially swiss army knives.   It's not really clear what they do or why.   It makes them harder to maintain and again harder to re-use.   This is why I took the approach of not just having it inherited (I could have had pages.page inherit the pages.qci_ui_tools (which may not be the best name)).   Using composition rather than inheritance is also a way of making it much more clear as to what code is doing what.